opae.spec

```diff
@@ -58,43 +59,18 @@ OPAE headers, tools, sample source, and documentation
 
 
 %prep
-%setup -q -n %{name}
+%setup -q -n %{name}-%{version}
 
 %build
 rm -rf _build
 mkdir _build
 cd _build
 
-%cmake .. -DCMAKE_INSTALL_PREFIX=/usr  -DOPAE_PRESERVE_REPOS=ON -DOPAE_BUILD_LEGACY=ON -B $PWD
-
-%make_build  opae-c \
-         bitstream \
-         xfpga \
-         modbmc \
-         opae-cxx-core \
-         hello_cxxcore \
-         board_a10gx \
-         board_n3000 \
-         board_d5005 \
-         fpgaconf \
-         fpgametrics \
-         fpgainfo \
-         userclk \
-         object_api \
-         hello_fpga \
-         hello_events \
-         bist_app\
-         fpga_dma_N3000_test\
-         fpga_dma_test\
-         opae-c++-utils\
-         opae-c++-nlb\
-         nlb0\
-         nlb3\
-         nlb7\
-         mmlink\
-         fpgad\
-         fpgad-api\
-         fpgad-vc\
+%cmake .. -DCMAKE_INSTALL_PREFIX=/usr  -DOPAE_PRESERVE_REPOS=ON -DOPAE_BUILD_LEGACY=ON -DOPAE_BUILD_SAMPLES=ON -B $PWD
```

Comment:
We need to specify python version as 3.9 as it defaults to 2.7.
```
-DOPAE_PYTHON_VERSION=3.9
```

Reply:
by default rpm creation  process uses python version 3.9 , no need to set python version in cmake

Reply:
This is to tell OPAE build system which Python to look for. This default used to be 2.7 but was changed to 3.6 before it was changed to 3

Reply:
Could the cmake infra auto detect the python version and bail when it is not supported ?  I am thinking of the use case where 3.8 would also work and the user would need to know to upgrade to 3.9 and to use 3.9 in the configure step

Reply:
By adding `-DOPAE_PYTHON_VERSION=3.9` we are essentially telling cmake to bail if the Python that is found is not 3.9.
That is why we should change it from 2.7 to 3.9. We could also omit the version from the call to `find_package` but I'd rather not and be explicit about the required Python version.